books/RayTracingTheNextWeek.html

```diff
@@ -2221,25 +2246,90 @@
             return emitted;
 
         return emitted + attenuation * ray_color(scattered, background, world, depth-1);
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
     }
     ...
 
     int main() {
         ...
 
-        // Render
+        point3 lookfrom;
+        point3 lookat;
+        auto vfov = 40.0;
+        auto aperture = 0.0;
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
+        color background(0,0,0);
```

Comment:
Ugh. I have no idea how things were working in my universe's timeline. Nice catch.